crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -151,6 +151,13 @@ impl BaseClient {
         let account_data = &extensions.account_data;
         if !account_data.is_empty() {
             self.handle_account_data(&account_data.global, &mut changes).await;
+            for room_id in account_data.rooms.keys() {
+                if !rooms.contains_key(room_id) {
+                    if let Some(events) = account_data.rooms.get(room_id) {
+                        self.handle_room_account_data(room_id, events, &mut changes).await;
```

Comment:
I'm not sure if we should add those rooms to the joined/left rooms of the `SyncResponse`
I don't know how to get the Membership here so we know if it's a `JoinedRoom` or `LeftRoom` instance.